modules/core/arrow-syntax/src/main/kotlin/arrow/syntax/collections/array.kt

```diff
@@ -1,59 +0,0 @@
-package arrow.syntax.collections
-
-import arrow.core.Option
-import arrow.core.Predicate
-import arrow.core.toOption
-
-@Deprecated(message = "`firstOption` is now part of the Foldable interface and generalized to all foldable data types")
```

Comment:
Should I let them deprecated or as here (deleted), because they are in Foldable and I don't see a problem deleting these. @pakoito @raulraja

Reply:
fair